demisto_sdk/commands/validate/validators/BC_validators/BC114_is_changed_or_removed_fields.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+from typing import Dict, Iterable, List
+
+from demisto_sdk.commands.common.constants import (
+    INTEGRATION_FIELDS_NOT_ALLOWED_TO_CHANGE,
+    GitStatuses,
+)
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class IsChangedOrRemovedFieldsValidator(BaseValidator[ContentTypes]):
+    error_code = "BC114"
+    description = "Ensure a pre-defined list of fields wasn't removed or modified in the integration yml."
+    rationale = "We wish to keep our behavior between different versions of the content item so we wish to enforce backwards compatibility breaking."
+    error_message = "The following fields were modified/removed from the integration, please undo:\n{0}"
+    related_field = "feed, isfetch, longRunning, longRunningPort, ismappable, isremotesyncin, isremotesyncout"
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(
+                    "\n".join(
+                        [
+                            f"The following fields were {action}: {', '.join(fields)}."
+                            for action, fields in altered_fields.items()
+                            if fields
+                        ]
+                    )
+                ),
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if (old_obj := content_item.old_base_content_object)
+            and (old_obj_data := old_obj.data)  # type: ignore[attr-defined]
+            and (
+                altered_fields := self.obtain_removed_fields(
```

Comment:
is the issue altering? or only removing. Seems like we shouldnt be able to go from a feed to not a feed, but we should be able to add these features. 
asking because you have `removed or modified` in the description

Reply:
Yea, modified it to work only on removing, good point!